### ISSUE TITLE
Insufficient Text Contrast of Footer Links

### ISSUE THREAD

**Issue by gmeben** [MESH-Research/knowledge-commons-profiles#569]
Area of Site: Homepage
URL: https://hcommons-test.org/

**Description**
Some of the hyperlinks in the footer don't have sufficient contrast against the background due to the opacity set on their parent container. 

**Solution**
Change `.footer-legal-nav .wp-block-navigation-item a` in `/assets/css/theme.css?ver=1.0.0` to `opacity: 0.8`
